app/views/layouts/auth.html.slim

```diff
@@ -0,0 +1,13 @@
+doctype html
+html
```

Comment:
```suggestion
html.layout-authentication
```
This one also 🙌

Reply:
👍🏾 gotcha! updated in https://github.com/aetheryna/google_scraper_ruby/pull/12/commits/c7199ca0f663bad636a030a80a1d7d43962a01c1